README.md

```diff
@@ -8,152 +8,225 @@
+[Fedora Silverblue](https://silverblue.fedoraproject.org/) discourage
+installation of software on the host (they mostly don't even have a *classic*
+package manager), and instead install software as/in containers.
+
+**Hmm, why is that so?**
+
+There are several advantages to this approach including higher reliability,
+easier upgrading and easier debugging. Fedora Magazine has nice articles about
+both [CoreOS](https://fedoramagazine.org/introducing-fedora-coreos/) and
+[Silverblue](https://fedoramagazine.org/what-is-silverblue/).
+
+**OK, what is then Toolbox good for?**
+
+Toolbox is a tool built on top of [Podman](https://podman.io) (which is a tool
+for managing [OCI](https://www.opencontainers.org/)-based containers) providing
+sane default values when creating/running containers (so-called *toolboxes*).
+These containers then *seamlessly integrate* with the rest of the operating
+system (e.g., environmental variables, home folder, shared memory, important
+locations in the root filesystem).
+
+Bear in might that Toolbox is not meant to be "safe" in terms of isolating your
+actions in the container from the host. It strives to extend your workflow
+rather than isolate it using the container technology.
+
+**Uh.., what?**
+
+Let's say you want to try to use `ansible` but you don't want to bloat your
+system (e.g., Fedora Workstation) or you can't really install it comfortably
+(e.g., Silverblue).
+
+Just create a container with `toolbox create`, enter it with `toolbox enter`
+(you should see the '⬢' symbol at the beginning of the command prompt) and
+type `sudo dnf install ansible` (the container is Fedora-based).
+
+When the installation is done you should have `ansible` prepared for use,
+without affecting the base operating system (if you leave the container,
+`ansible` will no longer be available until you return into it).
+
+And because your home folder is automatically mounted in the container, you
+can easily execute your already existing ansible playbooks.
+
+When you're done, you can simply exit the container by typing `exit` or
+pressing `Ctrl+D` and delete the container with `toolbox rm` and the system
+should be in the same state when you started (apart from files you may have
+created/edited/removed in your home folder or other locations shared with the
+host's filesystem).
+
+Also notice that you did not need to run Toolbox with `sudo` or as a
+priviledged user!
+
+**Sounds good! So, to use Toolbox I need one of those mentioned OSs?**
+
+No, you don't! This tool **doesn't require** using an OSTree-based system — it
+works equally well if you're running e.g. existing Fedora Workstation or
+Server. But if you haven't given Silverblue a try then you definitely should!
 
-Toolbox requires at least Podman 1.4.0 to work, and uses the Meson build
-system.
+## Documentation
 
-The following dependencies are required to build it:
-- meson
-- go-md2man
+https://docs.fedoraproject.org/en-US/fedora-silverblue/toolbox/
```

Comment:
It might be better to consolidate all the documentation in the upstream manuals. eg., we can link to the top-level `toolbox(1)` manual here, and maybe those of the other commands too.